pyproject.toml

```diff
@@ -9,7 +9,8 @@ version = "0.0.1"
 requires-python = ">= 3.10"
 dependencies = [
   "torch==2.7.1",
-  "huggingface_hub==0.32.4"
+  "huggingface_hub==0.32.4",
+  "safetensors~=0.4.3"
```

Comment:
Concerning https://github.com/andrewdalpino/UltraZoom/pull/1/files#r2255848685

Added dependency to pyproject.toml too, as you suggested @andrewdalpino